tests/test_hypothesis_plugin.py

```diff
@@ -63,7 +63,6 @@ class NumbersModel(pydantic.BaseModel):
         nonnegfloat: pydantic.NonNegativeFloat
 
     class JsonModel(pydantic.BaseModel):
-        json_any: pydantic.Json
```

Comment:
This was a duplicate entry from `MiscModel`, at

https://github.com/samuelcolvin/pydantic/blob/b7a8ef25c667b5dd4c4cd0b109c6625d1a57139a/tests/test_hypothesis_plugin.py#L27